pygmt/src/wiggle.py

```diff
@@ -94,6 +104,29 @@ def wiggle(self, data=None, x=None, y=None, z=None, **kwargs):
     """
     kwargs = self._preprocess(**kwargs)  # pylint: disable=protected-access
 
+    if kwargs.get("G") is not None:
+        msg = (
+            "The 'color' parameter has been deprecated since v0.8.0"
+            " and will be removed in v0.12.0. Use fill_positive/fill_negative"
+            " instead."
+        )
+        warnings.warn(msg, category=FutureWarning, stacklevel=2)
+
+    if (fill_positive or fill_negative) and kwargs.get("G") is not None:
+        raise GMTInvalidInput("Use either fill_positive/fill_negative or color.")
+
+    if fill_positive:
+        if kwargs.get("G") is not None:
+            kwargs["G"] = [kwargs["G"], fill_positive + "+p"]
```

Comment:
I don't get it. At line 115, you already raised an error when both fill_positive/fill_negative and color are given. So here, `kwargs.get("G")` must be `None`, right?

Reply:
You're right @seisman, that statement is not needed here.